src/components/incoming-data-menu/incoming-data-menu.html

```diff
@@ -22,8 +22,8 @@
     </button>
   </ion-list>
 
-  <ion-list *ngIf="type === 'bitcoinAddress'">
-    <ion-item-divider>{{'Bitcoin Address' | translate}}</ion-item-divider>
+  <ion-list *ngIf="type === 'bitcoinAddress' || type === 'ethereumAddress'">
+    <ion-item-divider>{{coin.toUpperCase() + 'Address' | translate}}</ion-item-divider>
```

Comment:
Missing space before "Address".
I'm not sure it works with pipe `translate` (check later).

Reply:
We would have to add to the translate.pot files BTC Address, ETH Address, etc...

Reply:
ACK!

Reply:
Added space before 'Address' -> changed to ' Address'